web-client/src/presenter/actions/StatusReportOrder/prepareStatusReportOrderAction.ts

```diff
@@ -20,6 +20,7 @@ export const prepareStatusReportOrderAction = ({
     state.statusReportOrder,
   );
 
+  const isCalendared = caseDetail.status === CASE_STATUS_TYPES.calendared;
```

Comment:
Could we use the entity's [isCalendared](https://github.com/ustaxcourt/ef-cms/blob/d758d9a5a7c018a06f84813a6a4978ceedf15097/shared/src/business/entities/cases/Case.ts#L1866) method?

Reply:
Got it!